test/autobahn-server.js

```diff
@@ -1,31 +1,12 @@
 'use strict';
 
-var WebSocket = require('../');
-var WebSocketServer = WebSocket.Server;
+const WebSocket = require('../');
 
-process.on('uncaughtException', function (err) {
-  console.log('Caught exception: ', err, err.stack);
-});
-
-process.on('SIGINT', function () {
-  try {
-    console.log('Updating reports and shutting down');
-    var ws = new WebSocket('ws://localhost:9001/updateReports?agent=ws');
-    ws.on('close', function () {
-      process.exit();
-    });
-  } catch (e) {
-    process.exit();
-  }
-});
-
-var wss = new WebSocketServer({port: 8181});
-wss.on('connection', function (ws) {
+const wss = new WebSocket.Server({port: 8181});
```

Comment:
On the other hand the Autobahn report generator also runs on `9001`. Maybe I'll add a runtime option for that while defaulting to `9001`.